Shared/Samples/Show viewshed from point on map/ShowViewshedFromPointOnMapView.swift

```diff
@@ -0,0 +1,202 @@
+            // Asynchronously adds that feature to the table.
+            try await featureCollectionTable.add(feature)
+            await performGeoprocessing(featureCollectionTable)
+        } catch {
+            self.error = error
+        }
+    }
+    
+    /// Contains the logic for the geoprocessing and passes the result on to another function to display.
+    /// - Parameter featureCollectionTable: Holds the tapped location feature
+    
+    private func performGeoprocessing(_ featureCollectionTable: FeatureCollectionTable) async {
+        let params = GeoprocessingParameters(executionType: .synchronousExecute)
+        // Sets the parameters spatial reference to the point tapped on the map.
+        params.processSpatialReference = featureCollectionTable.spatialReference
+        params.outputSpatialReference = featureCollectionTable.spatialReference
+        // Create a feature with the feature collection table which has the reference to the tapped location.
+        let geoprocessingFeature = GeoprocessingFeatures(features: featureCollectionTable)
+        // Sets the observation point to the tapped location.
+        params.setInputValue(geoprocessingFeature, forKey: "Input_Observation_Point")
+        // Creates geoprocessing job and kicks off process of getting viewshed for the point.
+        model.geoprocessingJob = model.geoprocessingTask.makeJob(parameters: params)
+        model.geoprocessingJob?.start()
+        // Get the result of the geoprocessing job asynchronously.
+        let result = await model.geoprocessingJob?.result
+        switch result {
+        case .success(let output):
+            if let resultFeatures = output.outputs["Viewshed_Result"] as? GeoprocessingFeatures {
+                processFeatures(resultFeatures: resultFeatures)
+            }
+        case .failure(let error):
+            // Sets error to be displayed.
+            self.error = error
+        case .none:
+            // This case should never execute.
+            break
+        }
+    }
+    
+    /// If the feature set is returned from the geoprocessing, it iterates through each feature and adds it to the
+    /// graphic overlay to display to the user.
+    /// - Parameter resultFeatures: Passes on the results of the geoprocessing so that they can be displayed.
+    
+    private func processFeatures(resultFeatures: GeoprocessingFeatures) {
+        if let featureSet = resultFeatures.features {
+            // Iterates through the feature set.
+            for feature in featureSet.features().makeIterator() {
+                // Creates the graphic for each feature's geometry.
+                let graphic = Graphic(geometry: feature.geometry)
+                // Sets the graphic on the overlay to display to the user.
+                model.resultGraphicsOverlay.addGraphic(graphic)
+            }
+        }
+    }
+}
+
+private extension ShowViewshedFromPointOnMapView {
+    private class Model: ObservableObject {
```

Comment:
```suggestion
   class Model: ObservableObject {
```
As the extension is already private, no need to repeat access level.